postgres/pom.xml

```diff
@@ -21,7 +22,17 @@
         <dependency>
             <groupId>com.google.cloud.sql</groupId>
             <artifactId>jdbc-socket-factory-core</artifactId>
-            <version>1.0.5-SNAPSHOT</version>
+            <version>1.0.4</version>
```

Comment:
Is there a reason why you aren't using the same version as above?

Reply:
Procedure is:
1. LGTM & merge to this PR
2. Update all versions to production push new PR, merge.
3. Push new artifact to maven central
4. update all code to SNAPSHOT again, except the sample., Push a new PR, merge
5. done.  Update java-docs-samples to a version of this sample.
6. Update the docs.  Announce it.

Reply:
In some places `1.0.4` is used and other places `1.0.5-SNAPSHOT` is used. Shouldn't you be consistent?

Reply:
1.0.5-SNAPSHOT would be better, it just wasn't showing up for some reason.  I'll fix it.